sdk/core/azure-core-amqp/src/test/java/com/azure/core/amqp/implementation/ActiveClientTokenManagerTest.java

```diff
@@ -29,6 +30,7 @@
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.Mockito.when;
 
+@Isolated("StepVerifier.setDefaultTimeout")
```

Comment:
What does this Isolated do?

Reply:
Nothing, just a visual indicator on why the test class is isolated,